guides/common/modules/proc_installing-the-dns-infoblox-module.adoc

```diff
@@ -13,7 +13,6 @@ DNS records are managed only in the default DNS view, it's not possible to speci
 ----
 # {foreman-installer} --enable-foreman-proxy-plugin-dns-infoblox \
 --foreman-proxy-dns true \
---foreman-proxy-dns-managed false \
```

Comment:
If you check [the relevant code](https://github.com/theforeman/puppet-foreman_proxy/blob/8d3c7fd63999bade3fe12835316f01f0d9285503/manifests/config.pp#L24-L29) you see that (since Foreman 2.0) `--foreman-proxy-dns-managed` is only respected if the provider is `nsupdate` or `nsupdate_gss`. Since the line below sets it to `infoblox`, it is redundant.

In other places it's set explicitly to true because the user may have changed it to false in a different scenario and it is safer to be explicit than to rely on some undefined state.